.gitignore

```diff
@@ -4,6 +4,3 @@ coverage.data
 .bundle
 Gemfile.lock
 pkg/*
-# Added so not to force others to use .rvmrc and .overcommit.yml files.
-.rvmrc
-.overcommit.yml
```

Comment:
Some projects that I contribute to did not want them so I added them to my global .gitignore file.
Similar to you not wanting my Gemfile.lock file - some projects want them and some don't.